package.json

```diff
@@ -181,7 +181,7 @@
     },
     {
       "path": "commerce-sdk-isomorphic-with-deps.tgz",
-      "maxSize": "410 kB"
+      "maxSize": "425 kB"
```

Comment:
Had to increase this bundle size limit because different Node versions actually result in different bundle size.